deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -52,6 +52,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                     <placements-select label="Placements" title="Placements" v-bind:selectoptions="placements" v-bind:assignpublicip="assignPublicIP"
                         showhelp="true" v-on:helpclick="placementsHelpClick" v-on:assignpublicipclick="selectpublicip"></placements-select>
                     <placements-help v-show="showPlacementsHelp" v-bind:data="placementsHelpData"></placements-help>
+                    <label-input label="Replacement Timeout" placeholder="2700" v-model="replacementTimeout" title="Time out for cluster replacement in seconds"></label-input>
```

Comment:
be consistent with unit in help and error message?

Reply:
I think unit should be in minutes for cluster replacement timeout.

Reply:
done